### PR TITLE
[PAT-1332] Migrate jitsi bugsnag to sentry

### PR DESCRIPTION
**Pull request by ivanjiang5628**
## Description
- [Jira Ticket](https://jira-jane.atlassian.net/browse/PAT-1332)

- Introduces Sentry error tracking integration (web app only).
- Removes bugsnag from the web app

### Release Risk Assessment
Low Risk

### CODE REVIEW

File: sentry.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable */
+import * as Sentry from "@sentry/react"
+import logger from './react/features/app/logger';
+
+const SENTRY_DSN_KEY = process.env.SENTRY_DSN || "https://8b81f2744c0a5a73d15aa61497cd50a0@o4505290921410560.ingest.sentry.io/4505783957192704"
```
Comment (ivanjiang5628) [inline sentry.js:5]: For testing purposes, I have added a hardcoded DSN. I will remove it later with a force push

File: react/features/app/components/ErrorBoundary.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+};
+
+initSentry();
+
+export const ErrorBoundary = (props: Props) => {
+    const { children } = props;
+
+    if (navigator.product === 'ReactNative') {
+        return children;
+    }
```
Comment (ivanjiang5628) [inline react/features/app/components/ErrorBoundary.js:22]: We don't want to initialize sentry in the ios app.

File: sentry.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+export const initSentry = () => {
+    if (navigator.product === 'ReactNative') {
+        return;
+    }
```
Comment (ivanjiang5628) [inline sentry.js:10]: We don't want to initialize sentry in the ios app.

File: react/features/app/components/ErrorBoundary.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+    return <SentryErrorBoundary fallback = { <p>An error has occurred</p> }>{children}</SentryErrorBoundary>;
```
Comment (kaeramirez) [inline react/features/app/components/ErrorBoundary.js:24]: Hey Ivan, quick question on this. Do we know what this will look like in the UI?

Comment (ivanjiang5628) [reply react/features/app/components/ErrorBoundary.js:24]: It's just a HOC/wrapper and It runs behind the scenes and won't have any direct impact on the UI.

File: sentry.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+export const notifySentry = (error)=>{
+    if (Sentry && window.hasSentryInitialized) {
+        Sentry.captureMessage(error)
+    }
+}
+
+export const initSentry = () => {
+    const releaseStage = process.env.NODE_ENV;
+    const version = process.env.APP_VERSION;
+    const SENTRY_DSN_KEY = process.env.SENTRY_DSN || "https://8b81f2744c0a5a73d15aa61497cd50a0@o4505290921410560.ingest.sentry.io/4505783957192704";
```
Comment (ivanjiang5628) [inline sentry.js:14]: For testing purposes, I have added a hardcoded DSN. I will remove it later with a force push after the terraform PR is ready

**Review by bitghost2m — APPROVED** [review]

**Pull request merged**